install.sh

```diff
@@ -3096,6 +2260,66 @@ XSD_FILES="share/doc/xsd/acct.xsd \
 
 CONTEXT_SHARE=$(find share/context/ -type f \( ! -iname "*.sh" ! -iname "SConstruct" \))
 
+#-------------------------------------------------------------------------------
+# PROMETHEUS
+#-------------------------------------------------------------------------------
+
+ALERTMANAGER_VENDOR_DIR='alertmanager'
+NODE_EXPORTER_VENDOR_DIR='node_exporter'
+PROMETHEUS_VENDOR_DIR='prometheus'
+if [ $ARCH = 'arm64' ]; then
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Double quote to prevent globbing and word splitting.